atd-vze/src/views/Crashes/Maps/CrashEditCoordsMap.js

```diff
@@ -180,31 +189,18 @@ class CrashEditCoordsMap extends Component {
           <div className="nav" style={navStyle}>
             <NavigationControl showCompass={false} />
           </div>
+          {/* add nearmap raster source and style */}
+          {!isDev && (
+            <>
+              <Source {...LOCATION_MAP_CONFIG.sources.aerials} />
+              <Layer {...LOCATION_MAP_CONFIG.layers.aerials} />
+              {/* show street labels on top of other layers */}
+              <Layer {...LOCATION_MAP_CONFIG.layers.streetLabels} />
+            </>
+          )}
           <Marker latitude={markerLatitude} longitude={markerLongitude}>
             <Pin size={40} color={pinColor} isDragging={isDragging} animated />
           </Marker>
-          <MapStyleSelector>
```

Comment:
This is the removed toggle between base layers that was not working. I recommend we backlog this via a new issue. @patrickm02L

Reply:
This should not have come out; I'm going to restore it.